lumigator/python/mzai/backend/backend/tests/api/routes/test_api_workflows.py

```diff
@@ -7,19 +7,22 @@
 
 from backend.main import app
 
+
 @app.on_event("startup")
 def test_health_ok(local_client: TestClient):
-        response = local_client.get("/health/")
-        assert response.status_code == 200
+    response = local_client.get("/health/")
+    assert response.status_code == 200
+
 
 def test_upload_data_launch_job(local_client: TestClient, dialog_dataset):
     response = local_client.get("/health")
     assert response.status_code == 200
 
-    create_response = local_client.post("/datasets",
-            data={},
-            files={"dataset": dialog_dataset, "format": (None, DatasetFormat.EXPERIMENT.value)},
-        )
+    create_response = local_client.post(
+        "/datasets",
```

Comment:
should we change to `/datasets/' re slack convo?

Reply:
+1 to that. I think the way `local_client` handles it changes nothing (as it gets a 307 redirect that it follows automatically) but given a simple curl would break without the ending slash it makes sense to be consistent here and avoid redirects